### ISSUE TITLE
Variable called before assigment in unit-1

### ISSUE THREAD

**Issue by ghosh-r** [huggingface/deep-rl-class#79]
In the section- 'The package_to_hub function', there is a case of one variable being called before it was assigned.

This is what is given in the notebook:

```python3
# Create the evaluation env
eval_env = DummyVecEnv([lambda: gym.make(env_id)])

# TODO: Define the name of the environment
env_id = 'LunarLander-v2'
```
Here `env_id` name is called before it is assigned, and leads to error.

This is what it should have been:

```python3
# TODO: Define the name of the environment
env_id = 'LunarLander-v2'

# Create the evaluation env
eval_env = DummyVecEnv([lambda: gym.make(env_id)])

```

**Comment by ghosh-r** [comment]
I implemented a solution in #80

**Comment by simoninithomas** [comment]
Thanks for the update I merged the PR #80 🤗
